code/modules/mob/living/simple_animal/eldritch_demons.dm

```diff
@@ -37,8 +37,6 @@
 /mob/living/simple_animal/hostile/eldritch/Initialize()
 	. = ..()
 	add_spells()
-	//by default
-	mind.add_antag_datum(/datum/antagonist/heretic_monster)
```

Comment:
Not adding this anymore?

Reply:
Mind isnt added on Initalize, so it just runtimes every time.

Reply:
So is it added after someone takes control of it?

Reply:
yes